vevestaX/vevesta.py

```diff
@@ -302,9 +303,35 @@ def __profilingReport(self, fileName):
              }
         ]
         profilingDataframe = pandas.DataFrame(data)
+
+        new_dataframe = pandas.DataFrame(
+            {"Field Name": ["Distinct", "Distinct (%)", "Missing", "Missing (%)", "Infinite", "Infinite (%)", "Mean",
+                            "Minimum", "Maximum", "Zeros", "Zeros (%)", "Negative", "Negative (%)",
+                            "Total Memory Size"]})
+
+        for col in self.__data.columns:
+            col_dict = {"Distinct": self.__data[col].nunique(),
+                        "Distinct (%)": self.__data[col].nunique() / self.__data.shape[0],
+                        "Missing": self.__data[col].isna().sum().sum(),
+                        "Missing (%)": (self.__data[col].isnull().sum().sum() * 100) / (
+                                self.__data[col].notnull().sum().sum() + self.__data[col].isnull().sum().sum()),
+                        "Infinite": np.isinf(self.__data[col]).values.sum(),
+                        "Infinite (%)": np.isinf(self.__data[col]).values.sum() / (
+                                (~self.__data[col].isna().sum().sum()) + self.__data[col].isna().sum().sum()),
+                        "Mean": self.__data[col].mean(),
+                        "Minimum": self.__data[col].min(),
+                        "Maximum": self.__data[col].max(),
+                        "Zeros": (self.__data[col] == 0).sum(),
+                        "Zeros (%)": (self.__data[col] == 0).sum() / self.__data.shape[0],
+                        "Negative": (self.__data[col] > 0).sum(),
```

Comment:
you are calculating postive values here..not negative...please test the code by calculating each of these values manually